.github/actions/setup-miniconda/action.yml

```diff
@@ -73,21 +73,21 @@ runs:
           echo "Print the available disk space for manual inspection"
           df -h
 
-          MINIMUM_AVAILABLE_SPACE_IN_GB=3
+          # 3GB can still fail some tests, so set the minimum requirement space to 4GB
```

Comment:
nit: This comment might be more appropriate for the PR description.  A future maintainer would wonder "Yes we're using 4GB. Why did he mention 3GB?"

Reply:
Ah ok, that makes sense :)